test/maps/azalias/polymorphic_helpers_test.go

```diff
@@ -154,3 +158,13 @@ func TestInterfaceNil(t *testing.T) {
 		t.Fatal("expected nil Aliases")
 	}
 }
+
+func TestInterfaceJSONNull(t *testing.T) {
+	props1 := TypeWithRawJSON{
+		AnyObject: azcore.NullValue[*any](),
+	}
+	b, err := json.Marshal(props1)
+	require.NoError(t, err)
+	require.Contains(t, string(b), `"anyObject":null`)
+	require.NotContains(t, string(b), `"anything":null`)
```

Comment:
Couldn't this be stricter because having `nil` value, `Anything` shouldn't be serialized?
```suggestion
	require.NotContains(t, string(b), "anything")
```